src/main/java/frc/robot/commands/shooter/PivotMove.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands.shooter;
+
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.RobotConfig.ShooterConfig;
+import frc.robot.subsystems.shooter.Shooter;
+
+public class PivotMove extends Command {
+  private Shooter m_shooter;
+  private double m_multiplier;
+  private double startTime;
+
+  public PivotMove(Shooter shooter, double multiplier) {
+    m_shooter = shooter;
+    m_multiplier = multiplier;
+
+    addRequirements(m_shooter);
+  }
+
+  @Override
+  public void initialize() {
+    startTime = Timer.getFPGATimestamp();
+    m_shooter.setAngle(MutableMeasure.ofBaseUnits(160 * m_multiplier, Units.Rotations));
```

Comment:
Don't instantiate a new MutableMeasure here. Make a constant for it in `RobotConfig`. Also yes, why a multiplier?

Reply:
added constant